### PR TITLE
Added exercises for the introduction, some videos are still missing

### PR DESCRIPTION
**Pull request by javier-romero**
Hi, 

I updated the web for the exercises of the intro.

Two questions:

- Could you give me access to the git at potassco-asp-course?
- Can I try the changes in my own computer? Using the commands of the README? I haven't tried yet (which also means that I can have some but in the request).

Chao!
Javier

### CODE REVIEW

**Comment by tortinator** [comment]
> Hi,
> 
> I updated the web for the exercises of the intro.
Muchas gracias! 
> Two questions:
> 
> * Could you give me access to the git at potassco-asp-course?

done: write access, please use pr :1st_place_medal: 

> * Can I try the changes in my own computer? Using the commands of the README? I haven't tried yet (which also means that I can have some but in the request).
>

Yes, this should work !

> Chao!
> Javier

File: introduction-exercises.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Introduction Exercises
+permalink: /introduction-exercises/
+---
+## Slides
+
+  [pdf](https://github.com/potassco-asp-course/exercises/releases/download/v1.0/introduction.pdf)
+  [git](https://github.com/potassco-asp-course/exercises)
+  [v1.0](ihttps://github.com/potassco-asp-course/exercises/tree/v1.0)
```
Comment (tortinator) [inline introduction-exercises.md:10]:
```suggestion
  [v1.0.0](https://github.com/potassco-asp-course/exercises/tree/v1.0)
```

File: index.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ The following slide deck and video playlist provide the last snapshot of the cur
  [videos](https://www.youtube.com/playlist?list=PL7DBaibuDD9NWhREiceTokOiY-S3nertB)
  [resources](/introduction/)
 
+ [slides](https://github.com/potassco-asp-course/exercises/releases/download/v1.0/introduction.pdf)
+ [videos](https://www.youtube.com/playlist?list=PL7DBaibuDD9PNXus6L5xzw7ATskhYKPby)
+ [resources](/introduction-exercises/)
+
```
Comment (tortinator) [inline index.md:73]: I lack a alternative but how shall we distinguis lectures and exercises..?

Also, why don't you use only 2 versioning numbers..?

File: introduction-exercises.md (hunk 1 above)
Comment (tortinator) [reply introduction-exercises.md:10]: Also: Does it ake sense to have both `intro.md` and `intro-ex.md` ...? Shouldn't these be one?

**Review by tortinator — COMMENTED** [review]
Hola @javier-romero , let's discuss my comments live :)

**Comment by tortinator** [comment]
Hola @javier-romero , I trust this is now a stale pull request, right..?

**Comment by javier-romero** [comment]
Yes, well closed :)

**Pull request closed without merging**